packages/sources/dxfeed-test/src/endpoint/price-router.ts

```diff
@@ -0,0 +1,93 @@
+import { RoutingTransport } from '@chainlink/external-adapter-framework/transports/meta'
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { batchTransport } from './price'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { wsTransport } from './price-ws'
+import { customSettings } from '../config'
+
+export const inputParameters = {
+  base: {
+    aliases: ['from', 'coin', 'market'],
+    type: 'string',
+    description: 'The symbol of symbols of the currency to query',
```

Comment:
I know this is a typo on the v3 framework types, but we're not extending it here, so we can fix it

```suggestion
    description: 'The symbol of the currency to query',
```